src/lib.rs

```diff
@@ -62,8 +62,8 @@ const ENV_CERT_FILE: &str = "SSL_CERT_FILE";
 
```

Comment:
Small typo in the commit message, "file must exists but mey be a non-regular" - "mey" :arrow_right:  "may"